docs/reference/linearmotor.md

```diff
@@ -23,6 +23,8 @@ It defaults to `"linear motor"`.
 - The `maxForce` field specifies both the upper limit and the default value for the motor *available force*.
 The *available force* is the force that is available to the motor to perform the requested motions.
 It is expressed in *newton* [N].
+When used in velocity mode, this maximum force is always applied (with the correct sign) until the speed is reached. 
+See : http://ode.org/wiki/index.php?title=Manual#Stops_and_motor_parameters
 The `wb_motor_set_available_force` function can be used to change the *available force* at run-time.
 The value of `maxForce` should always be zero or positive (the default value is 10 N).
-A small `maxForce` value may result in a motor being unable to move to the target position because of its weight or other external forces.
+A small `maxForce` value may result in a motor being unable to move to the target position because of its weight or other external forces. A motor can't go backward when used in speed mode.
```

Comment:
I'm not sure I understand this change. In velocity control positive or negative target velocities can be provided, what do you mean it can't go backwards?
Also, for documentation we usually write one sentence per line to avoid formatting manually. Same thing for the one below

Reply:
I mean that If when we request a movement in one direction, the resulting movement could not be in the opposite direction, whatever the force. For instance if we use request a vertical linear motor to extend (to push a mass to the top), even if the mass is crazy heavy, the motor could not retract. (which is actually not true for servomotor, when the external torque is too high, the servomotor go backward (or break...))
It's afaik, however, feel free to correct that :)

Reply:
Made a change, see if it better for you

Reply:
I understand what you mean now, but I'm not entirely convinced. See this example for instance:

https://user-images.githubusercontent.com/44834743/122745358-34049480-d289-11eb-946b-f1ccce14b137.mp4

Reply:
never meet this before ! 
so maybe remove this comment, better says anything than something wrong

Reply:
Yes maybe it's better to leave it out for now, but if you come across it again feel free to open an issue about it :)